CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Changelog
 
-## v2.14.2 (2020-04-30)
```

Comment:
Please do not remove chanelog entries :)

Reply:
doh